packages/client/hooks/useMediaRoom.ts

```diff
@@ -0,0 +1,37 @@
+import {useEffect, useReducer, useRef} from 'react'
+import initMediaRoom from '../utils/mediaRoom/initMediaRoom'
+import reducerMediaRoom from '../utils/mediaRoom/reducerMediaRoom'
+import {RoomStateEnum} from '../utils/mediaRoom/reducerMediaRoom'
+import useAtmosphere from './useAtmosphere'
+
+const initialState = {
+  mediaRoom: null,
+  room: {
+    activeSpeakerId: null,
+    state: RoomStateEnum.new
+  },
+  me: {
+    id: null,
+    deviceInfo: null
+  },
+  peers: {},
+  producers: {},
+  consumers: {}
+}
+
+const useMediaRoom = (roomId: string, teamId: string) => {
+  const atmosphere = useAtmosphere()
+  const {viewerId: peerId, authToken} = atmosphere
```

Comment:
is it safe to use the viewerId as the peerId? what if someone is logged in on 2 devices or 2 tabs?

Reply:
That's a good question. Assuming they log into the same Parabol account, what will currently happen is only one of the devices or tabs will be able to join the mediaroom. following the example of other video products, it seems we should let each of those clients join the room.

Any ideas on how to achieve it? The first thing that comes to mind is using a randomly generated id and prepending it to the viewerId.

Reply:
changed so that `peerId` is a composite of `clientId` and `viewerId`, where `clientId` is a randomly generated string for a single browser/device.